Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/DraftMessages/draftMessages.tsx

```diff
@@ -46,11 +46,12 @@ export interface IMessageState {
 
 class DraftMessages extends React.Component<IMessageProps, IMessageState> {
   private interval: any;
+  private openTaskModule: boolean;
```

Comment:
Change this and variable in messages.tsx to "isOpenTaskModuleAllowed"

Reply:
done